packages/core/src/Stores/client-store.js

```diff
@@ -693,12 +693,12 @@ export default class ClientStore extends BaseStore {
                     statement: 1,
                 })
             );
-            const account_settings = (await WS.authorized.storage.getSettings()).get_settings;
+            const account_settings = await WS.wait('get_settings').get_settings;
             if (account_settings && !account_settings.residence) {
                 await this.fetchResidenceList();
                 this.root_store.ui.toggleSetResidenceModal(true);
             }
-            WS.authorized.storage.landingCompany(this.residence).then(this.responseLandingCompany);
+            WS.wait('landing_company').then(this.responseLandingCompany);
```

Comment:
why doesn't storage work?

Reply:
@negar-binary we're not storing landing_company (or settings) in storage  - only these:

`socket-cache.js:`

```javascript
    const config = {
        payout_currencies: { expire: 120 },
        proposal_open_contract: { expire: 10 },
        active_symbols: { expire: 10 },
        contracts_for: { expire: 10 },
        exchange_rates: { expire: 60 },
        trading_times: { expire: 120 },
    };
```

we could also use the cache from deriv-api instead of storage - (since it's stored there):
`WS.authorized.cache.landingCompany(this.residence).then(this.responseLandingCompany);`

Or we could add `get_settings` and `landing_company` to storage?

Reply:
I think it makes sense to cache them in storage since they don't update regularly, what do you think?

Reply:
as discussed, lets go with the deriv-api cache.